osu.Game/Overlays/MusicController.cs

```diff
@@ -370,6 +367,9 @@ private void updateDisplay(WorkingBeatmap beatmap, TransformDirection direction)
                         currentBackground.Expire();
                         currentBackground = d;
                     }
+                })
+                {
+                    Depth = float.MaxValue,

```

Comment:
I wouldn't mind also setting other members like `Depth` automatically if we end up calling it `AsyncLoadWrapper`.